index.js

```diff
@@ -66,7 +66,55 @@ function toggleColor(element) {
         element.style.color = "green"
     }
 }
+const changeColor = document.querySelector("#header")
+changeColor.addEventListener('click', function (event){
+    // console.log(toggleColor)
+    if (changeColor.style.color === "green"){
+        changeColor.style.color = "black"
+    } else {
+        changeColor.style.color = "green"
+    }
+})
+     
 
 /***** Deliverable 2 *****/
+// const likeButton = document.querySelector("#profile > button")
+
+// likeButton.addEventListener('click', function(likes){
+
+
+// traveler.likes += 1 
+
+// likeCounter.innerHTML = `${traveler.likes} Likes`
+// })
+
+const likesButton = document.querySelector("#profile > button")
+function addLike (){
+    likeCounter = document.querySelector("#profile > p")
+    traveler.likes+=1
+    likeCounter.innerText = `${traveler.likes} Likes`
+}
+likesButton.addEventListener("click", addLike);
+//find likes in data.js 
+//update the likes 
+
 
 /***** Deliverable 3 *****/
+// const form = document.querySelector()  //find the form, set const, render the sighting
+const newForm = document.querySelector('form#new-animal-sighting-form')
+newForm.addEventListener('submit', function(event){
+    console.log('form submitted', event.target)
+    event.preventDefault()
+    
+    const speciesInput = event.target.species.value
+    const videoInput = event.target.link.value
+    const photoInput = event.target.photo.value
+    const decsInput = event.target.description.value
+    const newObject = {
```

Comment:
Great work on this last one!! If you wanted to get fancy w/ the ES6 syntax, you could make sure your variable names were the same names as the keys and do something like this...

    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
But I like your way and it's very readable and makes sense! Just showing you in case you see it done other ways.

PPS, what if we wanted to clear the form on a submit? There's probably a function on the event.target object that lets us do that... 🤔